tooling/api/src/tauri.ts

```diff
@@ -101,8 +101,8 @@ async function invoke<T>(cmd: string, args: InvokeArgs = {}): Promise<T> {
  * Note that `asset:` and `https://asset.localhost` must be added to [`tauri.security.csp`](https://tauri.app/v1/api/config/#securityconfig.csp) in `tauri.conf.json`.
  * Example CSP value: `"csp": "default-src 'self'; img-src 'self' asset: https://asset.localhost"` to use the asset protocol on image sources.
  *
- * Additionally, `asset` must be added to [`tauri.allowlist.protocol`](https://tauri.app/v1/api/config/#allowlistconfig.protocol)
- * in `tauri.conf.json` and its access scope must be defined on the `assetScope` array on the same `protocol` object.
+ * Additionally, `enable` must be set to `true` on [`tauri.security.assetProtocol`](https://tauri.app/v1/api/config/#securityconfig.assetprotocol)
+ * in `tauri.conf.json` and its access scope must be defined on the `scope` array on the same `assetProtocol` object.
```

Comment:
this change is in `next` branch not `dev`, did you mean to open this PR against next?